src/memory-analyzer/gdb_api.h

```diff
@@ -83,6 +83,12 @@ class gdb_apit
     const optionalt<std::string> string;
   };
 
+  /// Get the allocated size estimate for a pointer
```

Comment:
Could you explain why this is an estimate? And is it always either an under- or an overestimate?

Reply:
Added explanation: https://github.com/diffblue/cbmc/pull/4482/commits/ef457751a2867d1cb29b918ec98a2680bad80b24.